saas/settings.py

```diff
@@ -111,6 +111,12 @@
     'PROCESSOR_BACKEND_CALLABLE': None,
     'ROLE_RELATION': 'saas.Role',
     'TERMS_OF_USE': 'terms-of-use',
+    'AWS_ACCOUNT_ID': None,
```

Comment:
The `AWS_*` settings are not used anywhere in the code nor necessary. Only the `PICTURE_STORAGE_CALLABLE` should be here.

Reply:
We won’t be uploading picture to S3? Does this mean we should remove them from djaodjin-signup as well?

Reply:
Uploads to S3 is done through `django-storages` (optional). We need the `AWS_*` settings in djaodjin-signup to generate STS credentials, not to upload pictures.